.github/workflows/lint.yml

```diff
@@ -7,6 +7,8 @@ on:
     branches: [ $default-branch ]
   push:
     branches: [ $default-branch ]
+  workflow_dispatch:
```

Comment:
what is this about exactly? :)

Reply:
Ah, this would let us manually run our CI without needed to make a PR. Let me know if you prefer we remove this :)

https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

Reply:
Ah, this looked like a leftover. Admittedly, I didn't even _try_ to look what this is doing. Let's keep it!